lang/po/new/sv.po

```diff
@@ -1,1772 +1,1757 @@
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+
+# MSG_IR_03_OR_OLDER c=18
+#: messages.c:164
+msgid " 0.3 or older"
+msgstr " 0.3 el äldre"
+
+# MSG_FS_V_03_OR_OLDER c=18
+#: Marlin_main.cpp:9884
+msgid "FS v0.3 or older"
+msgstr "FS v0.3 el äldre"
+
+# MSG_IR_04_OR_NEWER c=18
+#: messages.c:163
+msgid " 0.4 or newer"
+msgstr " 0.4 el nyare"
+
+# MSG_FS_V_04_OR_NEWER c=18
+#: Marlin_main.cpp:9883
+msgid "FS v0.4 or newer"
+msgstr "FS v0.4 el nyare"
+
+# MSG_IR_UNKNOWN c=18
+#: messages.c:165
+msgid "unknown state"
+msgstr "okänt tillstand"
+
+# MSG_MEASURED_OFFSET c=20
+#: ultralcd.cpp:2913
+msgid "[0;0] point offset"
+msgstr "[0;0] punktförskjutn"
+
+# MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
+msgid "Crash detection can\\x0abe turned on only in\\x0aNormal mode"
+msgstr "Krockdetekt kan\\x0aendast aktiveras i\\x0anormalt läge"
+
+# MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
+msgid "WARNING:\\x0aCrash detection\\x0adisabled in\\x0aStealth mode"
+msgstr "VARNING:\\x0aKrockdetekt\\x0ainaktiverad i\\x0a tyst läge"
+
+# MSG_BABYSTEPPING_Z c=15
+#: ultralcd.cpp:3034
+msgid "Adjusting Z:"
+msgstr "Justerar Z:"
+
+# MSG_SELFTEST_CHECK_ALLCORRECT c=20
+#: ultralcd.cpp:8490
+msgid "All correct"
+msgstr "Allt korrekt"
+
+# MSG_WIZARD_DONE c=20 r=8
+#: messages.c:118
+msgid "All is done. Happy printing!"
+msgstr "Allt är klart. God utskrift!"
+
+# MSG_AMBIENT c=14
+#: ultralcd.cpp:1727
+msgid "Ambient"
+msgstr "Omgivande"
+
+# MSG_AUTO c=6
+#: messages.c:160
+msgid "Auto"
+msgstr "Auto"
+
+# MSG_PRESS c=20 r=2
+#: ultralcd.cpp:2485
+msgid "and press the knob"
+msgstr "och tryck pa knappen"
+
+# MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
+#: ultralcd.cpp:3355
+msgid "Are left and right Z~carriages all up?"
+msgstr "är bada Z-vagnarna helt uppe?"
+
+# MSG_AUTO_HOME c=18
+#: messages.c:11
+msgid "Auto home"
+msgstr "Auto hem"
+
+# MSG_AUTOLOAD_FILAMENT c=18
+#: ultralcd.cpp:6732
+msgid "AutoLoad filament"
+msgstr "Autoladda filament"
+
+# MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
+#: ultralcd.cpp:4317
+msgid "Autoloading filament available only when filament sensor is turned on..."
+msgstr "Autoladdning av fil. är endast möjligt när fil. sensorn är aktiv.."
+
+#: ultralcd.cpp:2648
+msgid "Autoloading filament is active, just press the knob and insert filament..."
+msgstr "Autoladdning filament är aktiv, tryck pa knappen och sätt i filament..."
+
+# MSG_SELFTEST_AXIS_LENGTH c=20
+#: ultralcd.cpp:8173
+msgid "Axis length"
+msgstr "Axellängd"
+
+# MSG_SELFTEST_AXIS c=16
+#: ultralcd.cpp:8174
+msgid "Axis"
+msgstr "Axel"
+
+# MSG_SELFTEST_BEDHEATER c=20
+#: ultralcd.cpp:8131
+msgid "Bed/Heater"
+msgstr "Bädd/Värmare"
+
+# MSG_BED_DONE c=20
+#: messages.c:15
+msgid "Bed done"
+msgstr "Bädd klar"
+
+# MSG_BED_HEATING c=20
+#: messages.c:16
+msgid "Bed Heating"
+msgstr "Bädden värms upp"
+
+# MSG_BED_CORRECTION_MENU c=18
+#: ultralcd.cpp:5798
+msgid "Bed level correct"
+msgstr "Bäddnivakorrigering"
```

Comment:
Baddplan? It is level as in flat, not as a height

Reply:
How to describe it in limit of 18 chars?
"Bäddytskorrigering" = 18